generated/java/proto-google-common-protos/src/main/java/com/google/api/AnnotationsProto.java

```diff
@@ -7,17 +7,6 @@
   private AnnotationsProto() {}
   public static void registerAllExtensions(
       com.google.protobuf.ExtensionRegistryLite registry) {
-    registry.add(com.google.api.AnnotationsProto.metadata);
```

Comment:
It looks like lots of stuff is being pulled out of annotations.proto. Is that what we expect to happen?